pkg/cmd/attestation/verification/extensions.go

```diff
@@ -18,19 +18,20 @@ func VerifyCertExtensions(results []*AttestationProcessingResult, ec Enforcement
 		return errors.New("no attestations proccessing results")
 	}
 
-	var atLeastOneVerified bool
+	var lastErr error
```

Comment:
To keep current tests passing, I return the last error found during verification. We can readdress this in another pull request if we want to return an error with different content.